net-libs/rest/rest-0.9.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson
+
+DESCRIPTION="Helper library for RESTful services"
+HOMEPAGE="https://wiki.gnome.org/Projects/Librest"
+
+LICENSE="LGPL-2.1"
+SLOT="3.1/0" # slot = libsoupslot.librest_api_version, subslot = librest_soversion in meson.build
+KEYWORDS="~amd64"
+IUSE="gtk-doc introspection test"
+RESTRICT="
+	!test? ( test )
+	gtk-doc? ( introspection )
```

Comment:
This should go in `REQUIRED_USE`, not `RESTRICT` (but the `test` stuff is correct).